### PR TITLE
Blitz test

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does
First round of converting the test to testNG

# Testing this PR

The "non-broken" tests should still run

# Related reading

https://trello.com/c/9nZfJ6Pi/132-import-static-cleanup

### CODE REVIEW

File: components/blitz/test/ome/formats/utests/DetectorSettingsTest.java [hunk 1]
```diff
@@ -75,27 +95,27 @@ public void testImageDetectorExists()
...
 	public void testContainerCount()
 	{
-	    assertEquals(1, store.countCachedContainers(Detector.class));
-	    assertEquals(1, store.countCachedContainers(Instrument.class));
-	    assertEquals(3, store.countCachedContainers(Pixels.class));
-	    assertEquals(5, store.countCachedContainers(null));
+	    Assert.assertEquals(1, store.countCachedContainers(Detector.class));
+	    Assert.assertEquals(1, store.countCachedContainers(Instrument.class));
+	    Assert.assertEquals(3, store.countCachedContainers(Pixels.class));
+	    Assert.assertEquals(5, store.countCachedContainers(null));
```
Comment (dominikl) [inline components/blitz/test/ome/formats/utests/DetectorSettingsTest.java]: Could be swapped to match arguments `assertEquals(int actual, int expected)`

File: components/blitz/test/ome/formats/utests/DetectorSettingsTest.java [hunk 2]
```diff
@@ -106,9 +126,9 @@ public void testReferences()
 	        LSID imageLsid = new LSID(Image.class, i);
 	        LSID dsLsid = new LSID(DetectorSettings.class, i,
 	                               CHANNEL_INDEX);
-	        assertTrue(store.hasReference(dsLsid, new LSID("Detector:0")));
-	        assertTrue(store.hasReference(imageLsid, new LSID("Instrument:0")));
+	        Assert.assertTrue(store.hasReference(dsLsid, new LSID("Detector:0")));
+	        Assert.assertTrue(store.hasReference(imageLsid, new LSID("Instrument:0")));
 	    }
-	    assertEquals(6, store.countCachedReferences(null, null));
+	    Assert.assertEquals(6, store.countCachedReferences(null, null));
```
Comment (dominikl) [inline components/blitz/test/ome/formats/utests/DetectorSettingsTest.java]: Same here

File: components/blitz/test/ome/formats/utests/DetectorSettingsTest.java (hunk 1 above)
Comment (jburel) [reply components/blitz/test/ome/formats/utests/DetectorSettingsTest.java]: yes, just noticed that too

**Comment by dominikl** [comment]
Looks good. Just can't run the tests locally, `unit.testng.xml` only runs a small subset of tests and `integration.testng.xml` fails with 
```
org.springframework.beans.BeanInstantiationException: Could not instantiate bean class [ome.services.scripts.ScriptRepoHelper]: Constructor threw exception; nested exception is ome.conditions.InternalException: Does not exist: /Users/dlindner/Repositories/sccmerges/openmicroscopy/components/blitz/./lib/scripts
```
However, Jenkins CI seems to be fine, no failed tests, so good to merge 👍

**Comment by jburel** [comment]
We will need to review the testing infrastructure since not all unit tests are run

**Pull request merged**